src/components/Navbar/WNavbar.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <header class="navbar">
+    <div class="logo-link">
+      <slot class="logo-link" name="logo"></slot>
+    </div>
+    <input id="side-menu" ref="checkbox" class="checkbox" type="checkbox" />
+    <nav class="navbar-tabs">
+      <ul class="navbar-links animated zoomIn">
+        <div class="close-menu-container" @click="closeMenu">
+          <CloseNavIcon />
+        </div>
+        <li v-for="link in navLinks" :key="link.key" class="nav-link" @click="closeMenu">
+          <a
+            :href="`#${link.key}`"
+            class="text font-[500] flex cursor-pointer sm:text-white-50 relative p-[8px] m-[-8px] pl-[16px]"
+            @mouseover="activateLink(link)"
+            @mouseleave="clearActiveLinks"
+          >
+            <Transition name="fade">
+              <NavigationCursor
+                v-if="link.active"
+                class="w-sm h-auto absolute top-[16px] left-[4px]"
+              />
+            </Transition>
+            <span /><span class="slash text-wit-blue-500">/</span>{{ link.locale }}
+          </a>
+        </li>
+        <!-- <li class="nav-link language-select">
+          <client-only>
+            <LanguageSwitcher />
+          </client-only>
+        </li> -->
+      </ul>
+    </nav>
+    <label ref="menu" for="side-menu" class="hamburguer-container">
+      <HamburgerIcon class="hamburguer" />
+    </label>
+  </header>
+</template>
+
+<script lang="ts" setup>
+import { ref, PropType, watch, computed } from 'vue'
+import { NavLink, NavLinks } from './WNavbar.ts'
+import NavigationCursor from '@/assets/svg/navigation-cursor.svg?component'
+import HamburgerIcon from '@/assets/svg/hamburguer.svg?component'
+import CloseNavIcon from '@/assets/svg/close-nav.svg?component'
+const menu = ref()
+const checkbox = ref()
+
+const props = defineProps({
+  navLinks: {
+    type: Object as PropType<NavLinks>,
+    required: true
+  },
+  redirectionActive: {
+    type: Boolean,
+    required: true
+  }
+})
+
+const localNabLinks = ref(props.navLinks)
+
+const redirectionActive = computed(() => props.redirectionActive)
+
+function closeMenu() {
+  menu.value.click()
+}
+
+function activateLink(link: NavLink) {
+  link.active = true
+}
+function clearActiveLinks() {
+  localNabLinks.value.forEach((link: NavLink) => (link.active = false))
+}
+
+watch(redirectionActive, () => {
+  if ((checkbox.value as HTMLInputElement).checked) {
+    closeMenu()
+  }
+})
+</script>
+
+<style lang="scss">
```

Comment:
Could we try to move the styles into tailwind classes?

Reply:
better to do it in a different pr